.gitignore

```diff
@@ -9,6 +9,7 @@ _build.log
 _install
 install
 bin/
+tests/
```

Comment:
should not ignore `tests` directory since our tests are in that directory.

Reply:
Good catch 👍